src/About/About.tsx

```diff
@@ -101,23 +106,16 @@ class About extends Component {
 				autoplayInterval={7000}
 				cellAlign="center"
 				slidesToScroll="auto"
-				renderBottomCenterControls={null}
-				renderCenterLeftControls={({ previousSlide }) => (
-					<Button mini variant="fab" onClick={previousSlide}>
-						<NavigateBeforeIcon />
-					</Button>
-				)}
-				renderCenterRightControls={({ nextSlide }) => (
-					<Button mini variant="fab" onClick={nextSlide}>
-						<NavigateNextIcon />
-					</Button>
-				)}
+				renderBottomCenterControls={undefined}
```

Comment:
Its because of the typings of this library. The don't support `null`. Only a value OR `undefined`. [Types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/nuka-carousel/index.d.ts#L178)